### ISSUE TITLE
Test speed/stdout

### ISSUE THREAD

**Issue by joshuap** [honeybadger-io/honeybadger-elixir#159]
Running the tests locally, they are generating a lot of stdout, and seem to be slower than I remember. Did this recently change, @minhajuddin?

**Comment by sorentwo** [comment]
If I recall this changed when the dummy phoenix app was integrated. At that point the suite got a fair bit slower and much noisier.

**Comment by minhajuddin** [comment]
We added a dummy phoenix app to improve our integration tests which added ~50 lines to stdout. However, it wouldn't have impacted the execution time for tests. The latest test suite took 4m33s on travis (https://travis-ci.org/honeybadger-io/honeybadger-elixir/builds/397492699) to run the whole matrix (10 combinations). I pulled an oldish build (https://travis-ci.org/honeybadger-io/honeybadger-elixir/builds/282520253) from 9 months ago and it took 3m27s. I think the integration test is valuable enough to have it at the expense of slow tests.
